test/acceptance/features/steps/steps.py

```diff
@@ -348,7 +348,12 @@ def operator_manifest_installed(context, backend_service):
         ns = context.namespace.name
     else:
         ns = None
-    _ = openshift.apply_yaml_file(os.path.join(os.getcwd(), "test/acceptance/resources/", backend_service + ".operator.manifest.yaml"), namespace=ns)
+
+    if backend_service == "backend":
+        _ = openshift.apply_yaml_file(os.path.join(os.getcwd(), "test/acceptance/resources/", backend_service + ".operator.manifest.yaml"), namespace=ns)
+        _ = openshift.apply_yaml_file(os.path.join(os.getcwd(), "test/acceptance/resources/backend_crd.yaml"), namespace=ns)
+    else:
+        _ = openshift.apply_yaml_file(os.path.join(os.getcwd(), "test/acceptance/resources/", backend_service + ".operator.manifest.yaml"), namespace=ns)
```

Comment:
let keep things simple - each fake operator is defined by single yaml file, i.e. all resources should be there, and not distributed across various files. In this way, the stop logic can be reverted to the previous state.

Reply:
Done.